src/redux/categories/categoriesSlice.js

```diff
@@ -1,16 +1,16 @@
-import { createSlice } from '@reduxjs/toolkit';
+// import { createSlice } from '@reduxjs/toolkit';
 
-const categorySlice = createSlice({
-  name: 'categories',
-  initialState: { categories: [], Status: 'Under Construction' },
-  reducers: {
-    getStatus: (state, action) => {
-      state.categories = action.payload === 'Under construction'
-        ? 'Under construction'
-        : state.categories;
-    },
-  },
-});
+// const categorySlice = createSlice({
+//   name: 'categories',
+//   initialState: { categories: [], Status: 'Under Construction' },
+//   reducers: {
+//     getStatus: (state, action) => {
+//       state.categories = action.payload === 'Under construction'
+//         ? 'Under construction'
+//         : state.categories;
+//     },
+//   },
+// });
 
-export const { getStatus } = categorySlice.actions;
-export default categorySlice.reducer;
+// export const { getStatus } = categorySlice.actions;
+// export default categorySlice.reducer;
```

Comment:
- [ ] @Kwamay Its not good practice having commented code in your codebase, kindly consider resolving issues in your categoriesSlcie instead of commenting out the whole file.